src/main/java/de/tum/in/www1/artemis/service/AssessmentService.java

```diff
@@ -229,12 +236,12 @@ public Result submitManualAssessment(long resultId, Exercise exercise) {
      * This function is used for saving a manual assessment/result. It sets the assessment type to MANUAL and sets the assessor attribute. Furthermore, it saves the result in the
      * database. If a result with the given id exists, it will be overridden. if not, a new result will be created.
      * <p>
-     * For programming exercises we use a different approach see {@link ProgrammingAssessmentService#saveManualAssessment(Result)}
+     * For programming exercises we use a different approach see {@link ProgrammingAssessmentService#saveManualAssessment(Result, User)}.
      *
-     * @param submission   the file upload submission to which the feedback belongs to
+     * @param submission   the submission to which the feedback belongs to
      * @param feedbackList the assessment as a feedback list that should be added to the result of the corresponding submission
-     * @param resultId     resultId of the submission we what to save the @feedbackList to, null if no result exists
-     * @return result that was saved in the database
+     * @param resultId     id of the result we what to save the feedbackList to, null if no result exists
```

Comment:
Still needs to be "want", right?
```suggestion
     * @param resultId     id of the result we want to save the feedbackList to, null if no result exists
```